spec.bs

```diff
@@ -198,6 +198,7 @@ dictionary AuctionAd {
   USVString buyerReportingId;
   USVString buyerAndSellerReportingId;
   sequence<USVString> allowedReportingOrigins;
+  sequence<DOMString> adRenderId;

```

Comment:
it's a DOMString, not a sequence https://crsrc.org/c/third_party/blink/renderer/modules/ad_auction/auction_ad.idl;drc=18cf8a61a0d39b5dd47a2f0163f2ecdaf29f6cc1;l=19

Reply:
Fixed.